### PR TITLE
source-facebook-marketing: 🎉 Add fields to the ads_set stream

### PR DESCRIPTION
**Pull request by mutt-santiago-stachuk**
### Add new fields to ads_set stream:
- bid_amount
- bid_constraints
- bid_strategy

## How

Add these fields in the `ads_set.json` schema of the `source-facebook-marketing` connector

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

### CODE REVIEW

**Review by leo-mansini — APPROVED** [review]
LGTM!

**Comment by alafanechere** [comment]
Thank you @mutt-santiago-stachuk for this contribution!

File: airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_sets.json
Comment (alafanechere) [inline airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_sets.json:98]: Can you define the schema of the `bid_constraints` object please?

**Review by alafanechere — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_sets.json
Comment (mutt-santiago-stachuk) [reply airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_sets.json:98]: Sure!

Comment (mutt-santiago-stachuk) [reply airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/schemas/ad_sets.json]: Done :tada:

**Review by alafanechere — APPROVED** [review]
LGTM. Waiting for a final approval from the connector team.

**Review by sherifnada — APPROVED** [review]

**Comment by alafanechere** [comment]
@mutt-santiago-stachuk you forgot to bump the connector version in the Dockerfile :) I'm merging now, and bumping + publishing from another PR.

**Pull request merged**
